helpers/smokeTestHelpers.js

```diff
@@ -255,20 +255,35 @@ const wait = ms => new Promise(resolve => {
   setTimeout(resolve, ms)
 })
 
-const validateWebhookReceived = async (env, paymentId) => {
+const retrieveWebhookObjectFromS3WithBackoffRetries = async (environment, resourceId) => {
```

Comment:
There is clear precedent for this pattern in this file but generally a named vs. anonymous is helpful for stack traces and debugging

```suggestion
async function retrieveWebhookObjectFromS3WithBackoffRetries (environment, resourceId) {
```

Reply:
Soo after slack discussion, and the stack trace showing nodejs gives us the var name in the stack trace we'll leave this as it is (especially to match the existing style in the file)